frontend/src/contexts/ContextProvider.js

```diff
@@ -0,0 +1,39 @@
+import { createContext, useContext, useState } from "react"
+
+const StateContext = createContext({
+    user: null,
+    token: null,
+    setUser: () => {},
+    setToken: () => {},
+
+})
+
+export const ContextProvider=({children})=>{
+    const [user,setUser]=useState({});
+    const [token,_setToken]=useState(null);     /*localStorage.getItem('ACCESS_TOKEN')*/
```

Comment:
For now let the comment stay but later we must get ride of this:  `/*localStorage.getItem('ACCESS_TOKEN')*/`